README.md

```diff
@@ -4,23 +4,23 @@
 
 This repo contains the source code for cross-platform [.NET Core](http://github.com/dotnet/core) command line toolchain. It contains the implementation of each command, the native packages for various supported platforms as well as documentation. 
 
-RTM and Preview 2 bits
----------------------
-To get the latest released bits (RTM for .NET Core and Preview 2 for tooling), 
-check out our [Getting started page](http://go.microsoft.com/fwlink/?LinkID=798306&clcid=0x409).
+Preview 4 release - MSBuild based tools
```

Comment:
@blackdwarf Should we add information about global.json and sdk version in case people want to run preview2 and preview4 side-by-side?